projectsCMSFunctions.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+//var_dump($_POST);
```

Comment:
?

Reply:
This is just an early draft of functions to process changes to the projects. Can ignore as not doing them now for MVP.

Reply:
remove var dump commented